template/webpack.config.develop.js

```diff
@@ -13,6 +14,11 @@ module.exports = webpackMerge(commonConfig, {
       aggregateTimeout: 300,
       poll: 1000
     }
-  }
+  },
 
+  plugins: [
```

Comment:
I think we should back this out, since I don't think will solve the issue for production builds, since this only happens for development, unless I'm missing something.